contracts/infrastructure/WalletFactory.sol

```diff
@@ -29,19 +29,22 @@ import "../modules/common/IVersionManager.sol";
  * @notice The WalletFactory contract creates and assigns wallets to accounts.
  * @author Julien Niset - <julien@argent.xyz>
  */
-contract WalletFactory is Owned, Managed {
+contract WalletFactory is Owned {
```

Comment:
I would prefer that as well, but it will require to add the `virtual` keyword to the methods of the `Managed` base contract. We would need to make sure it doesn't change the bytecode of the contracts that inherit from `Managed` .

Reply:
Adding `virtual` to `revokeManager` does change the bytecode, which is why I had initially decided against changing `Managed`.  But upon closer inspection, only one 256 bit word in the bytecode is changed, close to the end. This might be some sort of hash of the source code which would be inconsequential to the execution logic. Indeed I was able to verify in Etherscan the bytecode produced by using `virtual` with a source file that didn't use `virtual` and vice versa. So if Etherscan doesn't care about this difference in bytecode, I think we could ignore it as well and make the change to `Managed` without having to maintain some legacy `Managed` for our existing contracts.